chromadb/cli/cli.py

```diff
@@ -83,6 +88,45 @@ def run(
     uvicorn.run(**config)
 
 
+@app.command()  # type: ignore
+def vacuum(
```

Comment:
review: wondering if instead of exposing this at `chroma vacuum`, we should namespace and do something like `chroma maintenance vacuum`?

Reply:
a30d7a5b